test/test_bundle_importer.js

```diff
@@ -299,9 +299,14 @@ describe('Bundle Importer', function() {
   });
 
   describe('Changeset execution', function() {
-    it('Sets up the correct environment (Integration)', function() {
+    it('Sets up the correct environment (Integration)', function(done) {
       var data = utils.loadFixture(
           'data/wordpress-bundle-recordset.json', true);
+      bundleImporter.db.after('bundleImportComplete', function() {
```

Comment:
While unlikely it's possible that this `done()` is called before the assertions below. I'd recommend moving them into this callback.